src/pages/partners/_hero.tsx

```diff
@@ -55,7 +55,7 @@ const Hero = () => {
             >
                 <source data-src={PartnerVideo} type="video/mp4" />
             </StyledVideo>
-            <Content direction="column">
```

Comment:
Doesn't it affect the style?

Reply:
Not at all @mohammad-rework  This component not accept any css properties